src/foam/u2/wizard/data/UserCapabilityJunctionLoader.js

```diff
@@ -41,7 +41,7 @@ foam.CLASS({
           wizardlet.data = loadedData.clone(wizardlet.__subSubContext__);
         }
       } catch (e) {
-        console.log(e);
+        console.debug(e);
```

Comment:
are we leaving this in?  if so is debug appropriate? I would expect warning.

Reply:
so ya - i wasn't thinking about this well at the time. Appreciate the catch